src/input_adaptors/tables.py

```diff
@@ -0,0 +1,29 @@
+import json
+import os
+from input_adaptors.index import *
+from commands.keys import Keys
+
+
+class Tables:
+    def __init__(self, table_object):
+        self.table_object = table_object
+        self.name = table_object[Keys.NAME]
+        self.columns = table_object[Keys.COLUMNS]
+        self.primary_key = table_object[Keys.PRIMARY_KEY]
+        self.index = table_object[Keys.INDEX_KEYS]
+
+    def create_table(self, path):
+        self.validate(self)
```

Comment:
Why do you `validate` inside `serialize` method? You should validate inside creating the `TableMetadata` instance?